dGame/dUtilities/SlashCommandHandler.cpp

```diff
@@ -1231,6 +1238,18 @@ void SlashCommandHandler::HandleChatCommand(const std::u16string& command, Entit
 			return;
 		}
 
+		auto vehiclePhysicsComponent = newEntity->GetComponent<VehiclePhysicsComponent>();
+		if (vehiclePhysicsComponent) {
+			auto newRot = newEntity->GetRotation();
+			// Get the position and rotation of the player to spawn the vehicle
+			// and then invert it
+			// otherwise, we'll be upside down, thanks
+			auto angles = newRot.GetEulerAngles();
+			angles.x -= PI;
+			newRot = NiQuaternion::FromEulerAngles(angles);
+			newEntity->SetRotation(newRot);
+		}
+
```

Comment:
https://github.com/DarkflameUniverse/DarkflameServer/pull/726/files?w=0#diff-cac416f5d1a514ac99cc06f011e96aa580432cc53a090676084265ee3ccf326eR939-R949
This does not do the same work as above.  Is that intentional?

Reply:
that's debatable, they were the same as the code the comment refers to, but it was spawning and possessing it facing the opposite direction as the character was facing. Will make this the same as the other